app/Scripts/MainMenu.gd

```diff
@@ -18,5 +18,5 @@ func _ready():
 
 
 func _on_PlayButton_pressed():
-	get_parent().add_child(gameplay_scene.instance())
+	get_tree().change_scene("res://Scenes/Main.tscn")
```

Comment:
nit: can we remove line 3 then since it isn't used

('nit' is a github commenting convention, just means like a nit pick)

Reply:
Line 3?

Reply:
line 3 of this file i'm commenting on

Reply:
if you mean:
var main_menu_scene : PackedScene = load("res://Scenes/MainMenu.tscn")
It is being used , in line 13

Reply:
huh? I'm talking about `const gameplay_scene = preload("res://Scenes/Main.tscn")`